### PR TITLE
Mobileapps 970

### PR DESCRIPTION
**Pull request by aman-alfresco**
- added display for chips text
- added title for date picker calendar

### CODE REVIEW

File: search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt [hunk 1]
```diff
@@ -62,7 +62,13 @@ class ListViewFilterChips @JvmOverloads constructor(
     private fun String.take20orDefault(): String {
```
Comment (ankitgoyalgloballogic) [inline search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt:62]: Function name should be generic

File: search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt [hunk 2]
```diff
@@ -62,7 +62,13 @@ class ListViewFilterChips @JvmOverloads constructor(
...
         if (this.length <= 20)
```
Comment (ankitgoyalgloballogic) [inline search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt:63]: constant 20 should be in a variable

**Review by ankitgoyalgloballogic — CHANGES_REQUESTED** [review]

File: search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt (hunk 2 above)
Comment (aman-alfresco) [reply search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt:63]: it's default method name is take to get the string so I make for 20 character like this

File: search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt (hunk 1 above)
Comment (aman-alfresco) [reply search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt:62]: resolved

File: search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt (hunk 2 above)
Comment (aman-alfresco) [reply search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt:63]: resolved

**Review by gl-lovekesh — APPROVED** [review]

**Pull request merged**